mlflow/lightgbm.py

```diff
@@ -424,7 +429,32 @@ def log_feature_importance_plot(features, importance, importance_type):
             finally:
                 shutil.rmtree(tmpdir)
 
-        try_mlflow_log(log_model, model, artifact_path="model")
+        input_example = None
+        signature = None
+        try:
+            raw_data = train_data.get_data()
+            if isinstance(raw_data, str):
```

Comment:
i'm wondering if this is the best flow for this case (when the data is given as a path to a file)